src/distributions/wishart.jl

```diff
@@ -11,6 +11,12 @@ function Distributions.mean(::typeof(logdet), distribution::Wishart)
     return mapreduce(i -> digamma((ν + 1 - i) / 2), +, 1:d) + d * log(2) + logdet(S)
 end
 
+function Distributions.mean(::typeof(cholinv), distribution::Wishart)
+    d    = ndims(distribution)
+    ν, S = params(distribution)
+    sum([digamma.(0.5 * (ν + 1 - i)) for i in 1:d]) + d * log(2) + logdet(S)
```

Comment:
Replace `sum([ ... ])` with `sum(...)`. Note that I just removed squared brackets:
```julia
julia> foo(n) = sum([ i for i in 1:n ])
foo (generic function with 1 method)

julia> bar(n) = sum(i for i in 1:n)
bar (generic function with 1 method)

julia> @btime foo(100)
  97.671 ns (1 allocation: 896 bytes)
5050

julia> @btime bar(100)
  1.514 ns (0 allocations: 0 bytes)
5050
```

I also not sure about `.` in `digamma.`. It is a bit confusing, I suppose this code directly ported from ForneyLab. Probably we can simply omit `.`?

Reply:
Wow didn't know this

Reply:
@bartvanerp The reason is this :) 
```julia
julia> typeof((i for i in 1:10))
Base.Generator{UnitRange{Int64}, typeof(identity)}

julia> typeof([i for i in 1:10])
Vector{Int64}
```
So "bracket-free" version does not allocate and generates items lazily.